docs/getting_started/quick_start/fit_a_dihedral_with_Gaussian_and_write_all_files.rst

```diff
@@ -21,7 +44,7 @@ Import the required packages.
 .. code:: ipython3
 
     import unyt as u
-    from mosdef_dihedral_fit.dihedral_fit.fit_dihedral_with_gomc import fit_dihedral_with_gomc
+    #from mosdef_dihedral_fit.dihedral_fit.fit_dihedral_with_gomc import fit_dihedral_with_gomc
```

Comment:
should we just remove this line?

Reply:
I actually left it this way, thinking people will only copy each example.

Reply:
why are we removing it, do we not need it?